monai/handlers/tensorboard_handlers.py

```diff
@@ -87,18 +87,14 @@ class TensorBoardStatsHandler(TensorBoardHandler):
 
     """
 
-    @deprecated_arg("epoch_interval", since="1.1", removed="1.3")
-    @deprecated_arg("iteration_interval", since="1.1", removed="1.3")
     def __init__(
         self,
         summary_writer: SummaryWriter | SummaryWriterX | None = None,
         log_dir: str = "./runs",
         iteration_log: bool | Callable[[Engine, int], bool] = True,
         epoch_log: bool | Callable[[Engine, int], bool] = True,
```

Comment:
Looks like after removing `epoch_log` and `iteration_log` can also support `int`.

Reply:
If this is something to add I'd suggest raising an issue about it and keep this PR on the topic of removing things only.

Reply:
Sure, will do that, thanks!

Reply:
the `epoch_log`, `iteration_log` docstring currently don't state this feature clearly, the input could be boolean/int/callable...